docs/README.md

```diff
@@ -3,7 +3,7 @@
 * [Getting Started](getting-started.md) - How to get going with gulp
 * [API documentation](API.md) - Learn the ins and outs of using gulp
 * [CLI documentation](CLI.md) - Learn how to call tasks and use compilers
-* [Writing a Plugin](writing-a-plugin/README.md) - So you're writing a gulp plugin? Go here for the essential dos and don'ts.
+* [Writing a Plugin](https://github.com/gulpjs/gulp/tree/master/docs/writing-a-plugin) - So you're writing a gulp plugin? Go here for the essential dos and don'ts.
```

Comment:
No need to put the whole URL.  You can just remove the `README.md` part of the link